src/frontend/hooks/server/maps.ts

```diff
@@ -7,8 +7,11 @@ import {DOCUMENT_DIRECTORY} from '../../lib/file-system';
 
 import {createRefreshTokenStore} from '../refreshTokenStore';
 import noop from '../../lib/noop';
+import {ROOT_QUERY_KEY} from '../../constants';
 
-export const MAPS_QUERY_KEY = 'maps';
+export function getMapsQueryKey() {
+  return [ROOT_QUERY_KEY, 'maps'] as const;
+}
```

Comment:
also could add a comment here about where in core-react this comes from

also no need to export this:

```suggestion
function getMapsQueryKey() {
  return [ROOT_QUERY_KEY, 'maps'] as const;
}
```